cmat/trait_mapping/oxo.py

```diff
@@ -180,7 +180,7 @@ def get_oxo_results_from_response(oxo_response: dict) -> list:
             uri = str(oxo_mapping.uri)
 
             ontology_label = get_ontology_label_from_ols(uri)
-            if ontology_label is not None:
+            if ontology_label != '':
```

Comment:
I am wondering if we should do ontology_label.strip() just to be safe...

Reply:
No longer relevant as `None` to `''` change has been reverted.